.env

```
@@ -1 +1 @@
-VITE_API_BASE_URL = http://127.0.0.1:8080
\ No newline at end of file
+VITE_API_BASE_URL = http://192.168.1.19:8080
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)